docs/DeveloperGuide.adoc

```diff
@@ -424,46 +428,53 @@ Step 2. The user executes the command **tag i/1 t/Delicious** to tag the `**Tran
 TIP: `**ParserUtil**` is a class that contains useful functions for parsing the inputs from the user.
 
 [NOTE]
-Tags that already exist inside the specified `**Transaction**` will be ignored, and if that results in no tags being updated, an error will be shown to the user.
+Tags that already exist inside the specified `**Transaction**` will be ignored, and if that results in no tags being added, an error will be shown to the user.
 
-Since the user input is valid, a `**TagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object with the specified `**Tag**` inside replaces the original in the `**TransactionList**` at the same position.
+Since the user input is valid, a `**TagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object with the specified `**Tag**` added replaces the original in the `**TransactionList**` at the same position.
 
-image::tag-untag/TagUntagTransactionWTag.png[]
+image::tag-untag/TagUntag2.png[]
 
 Step 3. The user realises that the `**Tag**` added was not appropriate. The user then executes the command **untag i/1 t/Delicious** to untag the `**Tag**` "Delicious" from the `**Transaction**` at `**Index**` 1. The input is now checked with a `**UntagCommandParser**` parser and similarly, an attempt to parse each parameter occurs.
 
 * `**Index**` is parsed by `ParserUtil#parseIndex(String)`
 * `**Tag**` strings are parsed by `ParserUtil#parseTags(Collection<String>)` which calls `ParserUtil#parseTag(String)` iteratively for every string in the collection
 
 [NOTE]
-Tags that do not exist inside the specified `**Transaction**` will be ignored, and if that results in no tags being updated, an error will be shown to the user.
+Tags that do not exist inside the specified `**Transaction**` will be ignored, and if that results in no tags being deleted, an error will be shown to the user.
 
-Again, since the user input is valid, a `**UntagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object without the specified `**Tag**` replaces the original in the `**TransactionList**` at the same position.
+Again, since the user input is valid, a `**UntagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object with the specified `**Tag**` deleted replaces the original in the `**TransactionList**` at the same position.
```

Comment:
... with the specified deleted `**Tag**` replaces

Reply:
I don't see any issue with my current wording.